topics/releases.md

```diff
@@ -46,7 +46,7 @@ These may also add new features and introduce changes, but at a decreasing rate
 
 Once development has ended and the frequency of critical bug reports for the release candidate wanes, it is ready for the final release.
 
```

Comment:
is this space needed?

Reply:
I hope not - removed it and let's hope nothing major breaks :)